beacon_node/store/src/hot_cold_store.rs

```diff
@@ -1360,12 +1396,18 @@ impl<E: EthSpec, Hot: ItemStore<E>, Cold: ItemStore<E>> HotColdDB<E, Hot, Cold>
     pub fn get_blobs(&self, block_root: &Hash256) -> Result<Option<BlobSidecarList<E>>, Error> {
         let blobs_db = self.blobs_db.as_ref().unwrap_or(&self.cold_db);
 
+        // Check the cache.
+        if let Some(blobs) = self.block_cache.lock().get_blobs(block_root) {
+            metrics::inc_counter(&metrics::BEACON_BLOBS_CACHE_HIT_COUNT);
+            return Ok(Some(blobs.clone()));
+        }
+
         match blobs_db.get_bytes(DBColumn::BeaconBlob.into(), block_root.as_bytes())? {
             Some(ref blobs_bytes) => {
                 let blobs = BlobSidecarList::from_ssz_bytes(blobs_bytes)?;
-                // FIXME(sean) I was attempting to use a blob cache here but was getting deadlocks,
-                // may want to attempt to use one again
-                self.blob_cache.lock().put(*block_root, blobs.clone());
```

Comment:
Would be cool to test that this works fine on devnet now.

Reply:
working on it!

Reply:
Ok I've tested this out on the devnet now and in a local testnet and it seems to be working